Doc/library/functions.rst

```diff
@@ -635,19 +651,29 @@ are always available.  They are listed here in alphabetical order.
 
 .. function:: hex(x)
 
-   Convert an integer number to a lowercase hexadecimal string
-   prefixed with "0x", for example:
+   Convert an integer number to a lowercase hexadecimal string prefixed with "0x".
+   If x is not a Python :class:`int` object, it has to define an __index__() method
+   that returns an integer. Some examples:
 
       >>> hex(255)
       '0xff'
       >>> hex(-42)
       '-0x2a'
 
-   If x is not a Python :class:`int` object, it has to define an __index__()
-   method that returns an integer.
+   If you want to convert an integer number to uppercase or lowercase hexadecimal
+   string either with prefix or not, you can use either of the following ways.
+
 
-   See also :func:`int` for converting a hexadecimal string to an
-   integer using a base of 16.
+      >>> '%#x' % 255, '%x' % 255, '%X' % 255
+      ('0xff', 'ff', 'FF')
+      >>> format(255, '#x'), format(255, 'x'), format(255, 'X')
+      ('0xff', 'ff', 'FF')
+      >>> f'{255:#x}', f'{255:x}', f'{255:X}'
+      ('0xff', 'ff', 'FF')
+
+   See also :func:`format` for more information.
+
+   See also :func:`int` for converting a hexadecimal string to an integer using a base of 16.
```

Comment:
What I meant is an integer using base of 16, for eg., int('0xff', 16). If it is redundant, I can remove it.